addons/mail/static/src/js/mail.js

```diff
@@ -35,6 +35,20 @@ openerp.mail = function (session) {
             return [text, false];
         },
 
+        /* If there is unread message in thread, displays Mark all read button, else hide it.
+        */
+        show_mark_all_read : false,
```

Comment:
Ugh ? This should be a variable on the Wall or Thread manager, not on the chatter utilities ...